eclipse/che
# fixing tslint warnings

### What does this PR do?
https://github.com/eclipse/che/issues/21178

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/61005404/154532172-88fe5a19-4296-4986-9201-b0fce230a81d.png)


### What issues does this PR fix or reference?



### How to test this PR?

Execute the `npm test-git-ssh` command, so that below logs should be displayed
`[bmakam@localhost e2e]$ npm run test-git-ssh

> @eclipse-che/che-e2e@7.43.0-SNAPSHOT test-git-ssh
> ./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-git-ssh.json

Generating index.ts file...

> @eclipse-che/che-e2e@7.43.0-SNAPSHOT lint
> tslint --fix -p .


> @eclipse-che/che-e2e@7.43.0-SNAPSHOT tsc
> tsc -p .


################## Launch Information ##################

      TS_SELENIUM_BASE_URL: https://192.168.59.147.nip.io
      TS_SELENIUM_HEADLESS: false

      TS_SELENIUM_USERNAME: che@eclipse.org
      TS_SELENIUM_PASSWORD: admin

      TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME: petclinic-dev-environment
      TS_SELENIUM_DELAY_BETWEEN_SCREENSHOTS: 1000
      TS_SELENIUM_REPORT_FOLDER: ./report
      TS_SELENIUM_EXECUTION_SCREENCAST: false
      DELETE_SCREENCAST_IF_TEST_PASS: true
      TS_SELENIUM_REMOTE_DRIVER_URL: 
      DELETE_WORKSPACE_ON_FAILED_TEST: false
      TS_SELENIUM_LOG_LEVEL: INFO
      TS_SELENIUM_LAUNCH_FULLSCREEN: true

      to output timeout variables, set TS_SELENIUM_PRINT_TIMEOUT_VARIABLES to true
 ######################################################## 


  Git with ssh workflow
Can not get bearer token. URL used: https://192.168.59.147.nip.io/auth/realms/che/protocol/openid-connect/token
  [ERROR] PreferencesHandler.setPreferences failed to get user preferences: Error: unable to verify the first certificate
Can not get bearer token. URL used: https://192.168.59.147.nip.io/auth/realms/che/protocol/openid-connect/token
  [ERROR] PreferencesHandler.setPreferences failed to get user preferences: Error: unable to verify the first certificate
Can not get bearer token. URL used: https://192.168.59.147.nip.io/auth/realms/che/protocol/openid-connect/token
Error: unable to verify the first certificate`

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
